cli/cmd/service/create.go

```diff
@@ -17,10 +23,39 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			res1, err := client.CreateService(cmd.Context(), &adminv1.CreateServiceRequest{
+			// Parse attributes if provided
+			var attrs map[string]string
+			if attributes != "" {
+				if err := json.Unmarshal([]byte(attributes), &attrs); err != nil {
+					return err
+				}
+			}
+
+			req := &adminv1.CreateServiceRequest{
 				Name:             args[0],
 				OrganizationName: ch.Org,
-			})
+			}
+
+			// Set org role if provided
+			if orgRole != "" {
+				req.OrgRoleName = orgRole
+			}
```

Comment:
If org role is not set and if project role is not set, should we error? Or prompt for role?